db/migrate/20221213173527_add_institution_signature_to_collection.rb

```diff
@@ -0,0 +1,5 @@
+class AddInstitutionSignatureToCollection < ActiveRecord::Migration[6.0]
+  def change
+    add_column :collections, :institution_signature, :boolean, default: false
```

Comment:
Default to true.

Reply:
Could we just remove the `institution_signature` column completely because it won't be used anywhere?